contracts/tokenization/IERC20.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: MIT
```

Comment:
remove this file
create istakedtoken instead

Reply:
done 2f9cac0e9bfe9ef304fa27674892f2a2d5d82606